### PR TITLE
Add sponsors and about page

### PR DESCRIPTION
**Pull request by mermop**
- Add sponsors
- Add header and footer
- Add about page

Using data yml files for sponsor and nav.

Added a link back to home page from non-home pages.

Haven't tested on IE/edge.

<details>
<summary>Home page</summary>

screencapture-localhost-4567-1496190234691
screencapture-localhost-4567-1496190224650

</details>

<details>
<summary>About page</summary>

screencapture-localhost-4567-about-1496189538521
screencapture-localhost-4567-about-1496189555190
</details>

### CODE REVIEW

File: source/assets/public/index.css [hunk 1]
```diff
@@ -308,17 +311,67 @@ body {
     margin-top: 2rem;
   }
 
-.back-to-top {
```
Comment (mermop) [inline source/assets/public/index.css:311]: @libbyberrie this doesn't seem to be used - can I  check with you that it can be deleted

File: source/index.html.slim [hunk 2]
```diff
@@ -1,88 +1,42 @@
 ---
 title: Home
+home: true
 ---
 
-.container
```
Comment (mermop) [inline source/index.html.slim:5]: Have moved a lot of this to partials and layouts

File: source/about.html.slim [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  .body-inner-wrapper
+    .body-inner-wrapper__section#about
+      h2 📕 About Kiwi Ruby
+      p Kiwi Ruby is the first Ruby conference in New Zealand.
+      p Ruby is a friendly and interesting language, and the community is full of friendly and interesting people.
+      p We want to give the community the opportunity to learn and grow, and we want to showcase the talent we have right here in New Zealand.
+    .body-inner-wrapper__section#signup
+      h2 ✨ Diversity, inclusivity, and accessibility
+      p
+        ' We are committed to making Kiwi Ruby accessible and inclusive. We believe it's important that the community is diverse and can accommodate many different people.
```
Comment (mermop) [inline source/about.html.slim:15]: can you read over this copy @raquelxmoss ? Do you think we need a more detailed diversity/accessibility policy/statement?

File: data/sponsors.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+          - "We are proud to sponsor Kiwi Ruby to support our talented Technology team and help promote diversity in the local Ruby community, because we love to give."
+  - design:
+    logo_size: 1
+    name: "Design sponsor"
+    sponsors:
+      - name: "Icelab"
+        logo: "icelab.png"
+        website: "https://www.icelab.com.au"
+        description_paragraphs:
+          - "Icelab has been with us from Day One. From scheming in the back of the bus back from Rails Camp right through to the beautiful and sleek logo and website, Icelab has been a delight to work with, and their commitment to the Ruby community is legendary."
```
Comment (mermop) [inline data/sponsors.yml:24]: @libbyberrie I'll need to run this past y'all

File: source/assets/public/index.css [hunk 5]
```diff
@@ -54,6 +54,8 @@ body {
   text-rendering: optimizeLegibility;
   font-size: 1.8rem;
   line-height: 1;
+  color: var(--text-color);
+  font-family: var(--font-body);
```
Comment (mermop) [inline source/assets/public/index.css:58]: Setting these as defaults as eg `li` and `h3` items were coming up completely unstyled

File: source/assets/public/index.css [hunk 6]
```diff
@@ -186,35 +192,59 @@ body {
     vertical-align: middle;
   }
 
+/*Body text defaults*/
+
+p {
+  font-size: 2rem;
```
Comment (mermop) [inline source/assets/public/index.css:215]: Reorganising these defaults as the way they were organised before was overly specific and making it difficult to style child `p` elements differently. Happy to discuss

File: source/assets/public/index.css [hunk 7]
```diff
@@ -274,26 +293,10 @@ body {
...
 
-
-
-
 .footer {
   margin-bottom: 6rem;
-  width: 90%;
   text-align: center;
 }
-  .footer__nav {
```
Comment (mermop) [inline source/assets/public/index.css:285]: moved this to `nav--footer` as most of these styles are shared with the header nav

File: source/assets/public/index.css [hunk 8]
```diff
@@ -274,26 +293,10 @@ body {
   margin-top: -1rem;
 }
 
-
-
-
 .footer {
   margin-bottom: 6rem;
-  width: 90%;
```
Comment (mermop) [inline source/assets/public/index.css:282]: Why was this here? It was putting the centring off

File: data/nav.yml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+nav_items:
+  - title: "About"
+    link: "/about"
+  - title: "CFP"
+    link: "/#cfp"
```
Comment (mermop) [inline data/nav.yml]: Should this just link straight to papercall actually?

Comment (raquelxmoss) [reply data/nav.yml]: Yeah. Papercall has the most comprehensive info, and it puts them in the context of submitting (rather than making them click through to submit).

File: source/about.html.slim [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+      p We have a <a href="http://ruby.nz/code-of-conduct">code of conduct</a> in place. We will not tolerate harassment.
+      p We have confirmed with the venue that the stage and audience are accessible to people with mobility impairments. There will be an audio loop available for people with hearing impairments.
+      p There will be childcare available onsite.
+      p If you have any accessibility needs or concerns, please get in touch with us at <a href="mailto:kiwi@ruby.nz">kiwi@ruby.nz</a>.
+    .body-inner-wrapper__section#volunteer
+      h2 🤓 Organisers
+      p Kiwi Ruby is a community event run by volunteers, overseen by <a href="https://www.ruby.nz">Ruby New Zealand</a>.
+      p The organisers are Merrin Macleod, Raquel Moss, Nahum Wild, and Philip Arndt. Between us, we have run Rails Camps, Rails Girls events, and local meetups for many years.
+      p We thought it was time for New Zealand to have a Ruby conference. It became apparent that nobody else was going to do it, so we are doing it.
```
Comment (raquelxmoss) [inline source/about.html.slim]: ⚠️ I think this comes across as really snarky. What about "We thought it was time for New Zealand to have a Ruby conference, so we're making it happen"?

File: source/about.html.slim [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+      p
+        ' We are committed to making Kiwi Ruby accessible and inclusive. We believe it's important that the community is diverse and can accommodate many different people.
+      p We have a <a href="http://ruby.nz/code-of-conduct">code of conduct</a> in place. We will not tolerate harassment.
+      p We have confirmed with the venue that the stage and audience are accessible to people with mobility impairments. There will be an audio loop available for people with hearing impairments.
+      p There will be childcare available onsite.
+      p If you have any accessibility needs or concerns, please get in touch with us at <a href="mailto:kiwi@ruby.nz">kiwi@ruby.nz</a>.
+    .body-inner-wrapper__section#volunteer
+      h2 🤓 Organisers
+      p Kiwi Ruby is a community event run by volunteers, overseen by <a href="https://www.ruby.nz">Ruby New Zealand</a>.
+      p The organisers are Merrin Macleod, Raquel Moss, Nahum Wild, and Philip Arndt. Between us, we have run Rails Camps, Rails Girls events, and local meetups for many years.
```
Comment (raquelxmoss) [inline source/about.html.slim]: Links to our twitterz/other social media maybe?

File: source/about.html.slim (hunk 3 above)
Comment (raquelxmoss) [reply source/about.html.slim:15]: I think this is good

File: source/assets/public/index.css [hunk 12]
```diff
@@ -72,25 +74,29 @@ body {
...
+  text-align: right;
+}
+  .nav__item {
+    color: var(--primary-color);
+    display: inline;
+    font-family: var(--font-subtitle);
+    font-size: 1.6rem;
+    padding-left: 4rem;
+    text-decoration: none;
+  }
```
Comment (raquelxmoss) [inline source/assets/public/index.css:110]: From the diff it looks like this is indented one too many 🙀

File: source/assets/public/index.css [hunk 13]
```diff
@@ -308,17 +311,67 @@ body {
...
-  margin-left: auto;
-  margin-right: auto;
+/* Sponsorship section */
+
+.c-sponsor-level__name {
+  font-family: var(--font-heading);
+  color: var(--primary-color);
+  font-size: 3rem;
+  margin-bottom: 2rem;
+  margin-top: 2rem;
```
Comment (raquelxmoss) [inline source/assets/public/index.css]: Totally subjective design opinion: I reckon these could do with a bit more `margin-top`.

File: source/assets/public/index.css [hunk 14]
```diff
@@ -72,25 +74,29 @@ body {
...
   padding-left: 15rem;
 }
+
+/* Navigation */
+
+.nav {
+  line-height: 3rem;
+  padding-bottom: 1.5rem;
+  padding-right: 1rem;
+  text-align: right;
```
Comment (raquelxmoss) [inline source/assets/public/index.css]: It would be cool if we could sticky the nav

**Review by raquelxmoss — CHANGES_REQUESTED** [review]
Nice! Just one ⚠️ Blocker ⚠️, other comments totally optional and up to you 😄 👍

File: source/about.html.slim (hunk 10 above)
Comment (mermop) [reply source/about.html.slim]: You are not wrong

File: source/assets/public/index.css (hunk 12 above)
Comment (mermop) [reply source/assets/public/index.css:110]: I'm following the pattern @libbyberrie set of indenting `__element`s

File: source/about.html.slim (hunk 11 above)
Comment (mermop) [reply source/about.html.slim]: Phil's twitter is private so I don't know about linking to it and can't ask him at the moment - happy to link to others though

**Review by raquelxmoss — APPROVED** [review]
🆒 👍

File: source/assets/public/index.css (hunk 14 above)
Comment (mermop) [reply source/assets/public/index.css]: what do you think @libbyberrie

File: source/assets/public/index.css (hunk 1 above)
Comment (libbyberrie) [reply source/assets/public/index.css:311]: Whoops yeah. that's an artifact from when we were using the emoji for it

File: data/sponsors.yml (hunk 4 above)
Comment (libbyberrie) [reply data/sponsors.yml:24]: I'll put @timriley on this :)

File: source/assets/public/index.css (hunk 12 above)
Comment (libbyberrie) [reply source/assets/public/index.css:110]: Yeah sorry that's just to help me see what belongs to what. You're welcome to change it if it makes more sense to you the other way! :)

File: source/assets/public/index.css (hunk 14 above)
Comment (libbyberrie) [reply source/assets/public/index.css]: I think that could be a nice idea and would help keep it noticeable  when stickie'd you may also want to centre it or reduce it to 'hamburger' form which pops out on mobile.

File: source/assets/public/index.css (hunk 8 above)
Comment (libbyberrie) [reply source/assets/public/index.css:282]: I think initially it was to better centre it in relevance to the left-aligned paragraph but it doesn't need to be there.

File: source/assets/public/index.css (hunk 13 above)
Comment (libbyberrie) [reply source/assets/public/index.css]: Same here. It's looking a little crammed in - especially at `breakpoint-tablet` and up.

File: source/assets/public/index.css (hunk 8 above)
Comment (libbyberrie) [reply source/assets/public/index.css:282]: You can see what I mean when you compare the centring of the footer to the centring of the sponsored elements.

File: source/assets/public/index.css [hunk 15]
```diff
@@ -308,17 +311,67 @@ body {
...
-    .back-to-top:hover .body__subtitle {
-      color: var(--primary-color);
-    }
+.c-sponsor-level {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.c-sponsor-level--size-1 .c-sponsor-level__sponsor {
+  flex-basis: 100%;
```
Comment (libbyberrie) [inline source/assets/public/index.css:347]: I'm not sure what the best way to do this would be, but it may be a good idea to reduce max-dimensions on the sponsor containers - because on larger screen it almost looks comically large (though by the looks of it the dimensions will change when other sponsors are added in so my opinion may change there!

**Comment by libbyberrie** [comment]
Most of it is looking really good! One comment about the nav though: 
image
It took me a good couple of minutes to figure out where the nav was on my monitor. A way to reduce this may be that in `breakpoint-desktop-up` you add padding to the top/right hand sides to at least make it stick out a little more.

I also feel like pushing may not have happened cause it has a `nav--header` and `nav--footer were mentioned in your comments but i can't find those class' styles in the repo.

**Comment by mermop** [comment]
Oh I think I didn't actually put in any separate styles for `nav--header` and `nav--footer` now that I look at it.

Possibly stickying/adding a drop shadow would address that problem on desktop, although we then get the problem of doing a mobile nav which is always a pain

File: data/sponsors.yml (hunk 4 above)
Comment (mermop) [reply data/sponsors.yml:24]: Any feedback @libbyberrie @timriley ?

Comment (timriley) [reply data/sponsors.yml:24]: @mermop that is flattering and wonderful! Happy for you to share that :)

**Comment by mermop** [comment]
I've given the header nav a max width. I think we probably also want an overall max width for the content for wide screens. 

Once we have a few more pages on the menu we may want to think about hamburgering.
